src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2222,6 +2222,33 @@
      * A list of resources in the Java 7 try-with-resources construct.
      * This is a child of RESOURCE_SPECIFICATION.
      *
+     * <p>For example:</p>
+     * <pre>try (FileReader fr = new FileReader("config.xml")) {}</pre>
+     * <p>parses as:</p>
+     * <pre>
+     * --LITERAL_TRY -&gt; try
```

Comment:
Remove leading '--' to indent code to the left.
Please use 'finally {}' instead of "catch" and include it in ast print in javadoc